File: v3/node/transaction/orders/modify_order.js
const FyersAPI = require("fyers-api-v3").fyersModel


var fyers = new FyersAPI({ path: "/path/to/where/logs/to/be/saved" })
// set appID
fyers.setAppId("Qxxxxxx75-1xx")

// set redirectURL
fyers.setRedirectUrl("https://XXXXX.com")

// set accessToken
fyers.setAccessToken("eyJ0xxxx")

var inp={
    "id":'808058117761', 
    "type":1, 
    "limitPrice": 61049, 
    "qty":1
}

fyers.modify_order(inp).then((response) => {
    console.log(response)
}).catch((error) => {
    console.log(error)
})

File: v3/python/transaction/orders/modify_order.py
from fyers_apiv3 import fyersModel

client_id = "XC4XXXXM-100"
access_token = "eyJ0eXXXXXXXX2c5-Y3RgS8wR14g"

# Initialize the FyersModel instance with your client_id, access_token, and enable async mode
fyers = fyersModel.FyersModel(client_id=client_id, token=access_token,is_async=True, log_path="")


orderId = "8102710298291"
data = {
    "id":orderId, 
    "type":1, 
    "limitPrice": 61049, 
    "qty":1
}

response = fyers.modify_order(data=data)
print(response)